09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>This whole section can be shortened by using Freud to compute the MSD: <a href="https://freud.readthedocs.io/en/latest/modules/msd.html" target="_blank">https://freud.readthedocs.io/en/latest/modules/msd.html</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>